src/modules/organisations/organisations.service.ts

```diff
@@ -21,17 +19,14 @@ import { CreateOrganisationMapper } from './mapper/create-organisation.mapper';
 import { UpdateOrganisationDto } from './dto/update-organisation.dto';
 import { OrganisationMembersResponseDto } from './dto/org-members-response.dto';
 import { OrganisationMemberMapper } from './mapper/org-members.mapper';
-import { OrganisationMember } from './entities/org-members.entity';
 
 @Injectable()
 export class OrganisationsService {
   constructor(
     @InjectRepository(Organisation)
     private readonly organisationRepository: Repository<Organisation>,
     @InjectRepository(User)
-    private readonly userRepository: Repository<User>,
-    @InjectRepository(OrganisationMember)
-    private readonly organisationMemberRepository: Repository<OrganisationMember>
```

Comment:
Please revert this back too

```suggestion
    @InjectRepository(OrganisationMember)
    private readonly organisationMemberRepository: Repository<OrganisationMember>
```